docs/admin/installation/hardware.rst

```diff
@@ -449,7 +449,33 @@ NUC before it can be used. We recommend:
 -  1x memory kit of compatible 2x8GB sticks
    -  You can put one 8GB memory stick in each of the servers.
 
-.. _nucc12_recommendation:
```

Comment:
Note that there was a typo in this anchor but I think it's fine to fix

Reply:
Thanks! Looks like it's fixed in the latest version of the branch.